docs_src/getting-started/Ch-GettingStartedSDK-Go.md

```diff
@@ -1,129 +1,92 @@
-command (a.k.a. actuation) requests. Finally, the framework provides the
-common code to help get the data coming from the sensor into EdgeX Core
-Data (often referred to as data ingestion). With the SDK, developers are
-left to focus on the code that is specific to the communications with
-the device via the protocol of the device.
-
-In this guide, you create a simple Device Service that generates a
-random number in place of getting data from an actual sensor. In this
-way, you get to explore some of the framework and work necessary to
-complete a Device Service without actually having a device to talk to.
+In this guide, you create a simple device service that generates a
+random number as a means to simulate getting getting data from an actual device. In this way, you explore some the SDK framework and work necessary to complete a device service without actually having a device to talk to.
 
 ## Install dependencies
 
-Creating a Device Service requires a little programming in Go. Go Lang
-(version 1.11 or better) must be installed on your system to complete
-this lab. Follow the instructions in the link below to install Go if it
-is not already installed on your platform:
-
-> <https://golang.org/doc/install>
-
-You need a Git tool to pull the Device Service Go SDK code from the
-EdgeX Foundry Git repository. Follow the instructions in the link below
-to install Git for your platform:
-
-> <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>
-
-You will also need a "make" program. On Ubuntu Linux environments, this
-can be accomplished with the following command:
-
-    sudo apt install build-essential
-
-Finally, you need a simple text editor (or Go Lang IDE).
+See the [Getting Started - Go Developers](Ch-GettingStartedGoDevelopers.md) guide to install the necessary tools and infrastructure needed to develop a GoLang service.
 
 ## Get the EdgeX Device SDK for Go
 
-Complete the following steps to create a folder on your file system,
-download the [Device SDK](../../microservices/device/sdk/Ch-DeviceSDK), then you
-pull the SDK to your system, and finally create the new EdgeX Device
-Service from the SDK templating code.
-
-1.  Create a collection of nested folders,
-    ~/go/src/github.com/edgexfoundry on your file system. This folder
-    will eventually hold your new Device Service. In Linux, this can be
-    done with a single mkdir (with -p switch) command:
-
-        mkdir -p ~/go/src/github.com/edgexfoundry
+Follow these steps to create a folder on your file system,
+download the [Device SDK](../../microservices/device/sdk/Ch-DeviceSDK), and get the GoLang device service SDK on your system.
 
-2.  In a terminal window, change directories to the folder you created:
+1.  Create a collection of nested folders, ~/go/src/github.com/edgexfoundry on your file system. This folder will hold your new Device Service. In Linux, create a directory with a single mkdir command
```

Comment:
inline code on the path

Reply:
i **REALLY** don't think we should be recommending to place services based on our SDKs inside the GOPATH -- this was recommended before modules were introduced. Additionally i dont think they should have to clone the SDK to use it? this is messy: 
` git clone --depth 1 --branch v1.2.2 https://github.com/edgexfoundry/device-sdk-go.git`

This should be something more along the lines 

1) create a directory for your new device services where you prefer

2) run `go mod init github.com/{your_username}/mynewdeviceservice`

3) create a `main.go `file and copy and paste the following code into it.
``` go
package main

import (
     "github.com/edgexfoundry/device-sdk-go"
)
func main(){

}
```
4) run `go build` and the SDK will be pulled down for you.



Though after reading through this---i see you are after the examples and copying that out. Definitely thinking we need to move it out/in addition place all required "getting started" code here in the docs to copy and paste

Reply:
I am not disagreeing with the input, but would like to get the examples functional again first and then we can try to tackle the better way.  Took me a couple of days to work with the SDK teams to have them realize broken elements and fixes to things like the device profile.  If you prefer, let's create an issue in the docs to go back and fix this once we have the rest of the clean up done.